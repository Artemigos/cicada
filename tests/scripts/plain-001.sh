plain_counter=35
echo hi
echo mid > /dev/null

echo hello \
       world

echo foo\
bar

echo hello world \
    | sed 's/hello/hi/' \
    | wc \
    | sed -e 's/  */ /g' \
          -e 's/^  *//'

echo bye "counter $plain_counter"
