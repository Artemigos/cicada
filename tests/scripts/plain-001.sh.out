hi
hello world
1 2 9
bye counter 35
